templates/buyer/get_discount.html

```diff
@@ -33,24 +39,30 @@ <h4 class="modal-title">Get Discount</h4>
 					</div>
 				</div>
 				<div class="modal-footer">
-					{% if form.errors %}
-						<button class="eds-btn eds-btn--button eds-btn--fill" type="button" onclick="window.open('{{ event.url }}', '_blank')">
-						Continue without discount
-						</button>
-						<input type="submit" class="eds-btn eds-btn--button eds-btn--fill" value="Retry"/>
-					{% else %}
-						<input type="submit" class="eds-btn eds-btn--button eds-btn--fill" value="OK"/>
-					{% endif %}
+					<button class="eds-btn eds-btn--button eds-btn--fill" id="retry" type="button" onclick="window.open('{{ event.url }}', '_blank')" style="display: none;">
+					Continue without discount
+					</button>
+					<input type="submit" id="submit_btn" class="eds-btn eds-btn--button eds-btn--fill" value="OK"/>
 				</div>
 			</div>
 		</form>
 	</div>
-	 <!-- <script>
+	 	<script>
```

Comment:
Please, try to move this script to a .js file and use it here like `<script srs="forms_options.js">`